quickget

```diff
@@ -939,14 +939,20 @@ function web_get() {
       exit 1
     fi
 
+    if wget --help | grep -q Wget2; then
```

Comment:
if wget -V | grep -q Wget2; then

Reply:
I think it's actually better to check whether wget2 is installed in the first place and use it if so; some distributions may not ship this package that symlinks /usr/bin/wget to /usr/bin/wget2. Wget2 is faster and should be used instead whenever available.

Reply:
Also, just remembered, the best way to check whether /usr/bin/wget leads to wget2 would be to use `readlink /usr/bin/wget` and check whether the output is equal to 'wget2'. That would be much faster than any other way, it only takes 2 milliseconds on my system compared to 6 with wget2's help and version menus.